src/options.js

```diff
@@ -89,4 +89,7 @@ export default {
   unsentKey: 'amplitude_unsent',
   unsentIdentifyKey: 'amplitude_unsent_identify',
   uploadBatchSize: 100,
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
+  },
```

Comment:
Since this option is a nested object, will a user passing in their own headers completely overwrite this (and therefore delete this `Content-Type` value)? Or will it keep Content-Type and append new headers from there?

Reply:
Also, don't forget to put a comment for `headers` at the description of all the possible options at the top of this file

Reply:
Added docs explaining this